### PR TITLE
Support specify cluster_id without prefix, Fix #88

### LINKED ISSUE #88: Possibility to specify cluster_id without prefix
The current implementation of **user-defined cluster_id** (https://github.com/ocp-power-automation/ocp4-upi-powervm/issues/68) looks a bit not totaly achieved to me (but it is maybe just for me...)

In all openshift documentation, `cluster_id` is a subdomain. 
API: api.<cluster_id>.<domain>  ex: api.mycluster.mydomain.org
applications: <app>.apps.<cluster_id>.<domain>  ex portal.apps.mycluster.mydomain.org

With the current implementation #69 you can't really specify your cluster_id ... it is still a concatenation of `cluster_id` , `cluster_prefix`.... (with a - in between).

I think it could be good to let the user been able to choose its cluster name (without "-") 

your thoughts?

### PR DESCRIPTION
**Pull request by cs-zhang**
Just use cloud_id if cloud_id_prefix is not set.

Signed-off-by: CS Zhang <zhangcho@us.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/187#" title="Author self-approved">cs-zhang</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **bpradipt** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @bpradipt` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yussufsh — APPROVED** [review]
/lgtm
As `cluster_id_prefix` has a default value of "test-ocp", user will always need to explicitly set this to "" to truly use this cluster_id only feature.

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/187#" title="Author self-approved">cs-zhang</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/187#pullrequestreview-612863358" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by yussufsh** [comment]
@cs-zhang We will need same change in powervs and kvm projects once this is merged. Thanks.

**Comment by bpradipt** [comment]
> /lgtm
> As `cluster_id_prefix` has a default value of "test-ocp", user will always need to explicitly set this to "" to truly use this cluster_id only feature.

better to document this in the variables file itself to avoid any confusion

**Comment by ltccci** [comment]
New changes are detected. LGTM label has been removed.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/187#pullrequestreview-613475932" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/187#" title="Author self-approved">cs-zhang</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/187#pullrequestreview-612863358" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
